wear/src/main/java/io/homeassistant/companion/android/conversation/views/ConversationView.kt

```diff
@@ -118,14 +116,7 @@ fun ConversationResultView(
 ) {
     val scrollState = rememberScalingLazyListState()
 
-    Scaffold(
-        positionIndicator = {
-            if (scrollState.isScrollInProgress) {
-                PositionIndicator(scalingLazyListState = scrollState)
-            }
-        },
-        timeText = { TimeText(scalingLazyListState = scrollState) }
-    ) {
+    Scaffold {

```

Comment:
Because the `ThemeLazyColumn` is updated to include a `Scaffold` here you have nested Scaffolds, don't think that is correct.

Reply:
I think we need this code in the `Scaffold` but it did load so I thought it was a loading based on a hierarchy

Reply:
Which `Scaffold` are you talking about? This is getting confusing

Reply:
I am talking about the specific `Scaffold` code here, the `LaunchedEffect` cannot be moved to the `ThemeLazyColumn` so while there is a nested `Scaffold` I think thats the only way to get it to work?

Reply:
But the `LaunchedEffect` can still be outside the `ThemeLazyColumn`?

```kotlin
val scrollState = rememberScalingLazyListState()
LaunchedEffect(key) { ... }
if (hapticFeedback) { ... }
ThemeLazyColumn(state = scrollState) { ... }
```

Reply:
oh my bad thought it had to be in the `Scaffold` but looks like it just needed a composable scope :)

Reply:
done, thank for the suggestion :)